src/components/Documentation/Card/index.tsx

```diff
@@ -11,15 +11,16 @@ interface Props {
   links: LinkItem[]
   title: string
   Icon: React.ComponentType<{ className?: string }>
+  anchorId: string
```

Comment:
I think you can make the anchorId optional, and then compute the href below depending if there is or not an anchorId.